cas-server-core-services/src/main/java/org/jasig/cas/authentication/principal/AbstractWebApplicationService.java

```diff
@@ -21,7 +21,7 @@
 
     private static final long serialVersionUID = 610105280927740076L;
 
-    private static final Map<String, Object> EMPTY_MAP = Collections.unmodifiableMap(new HashMap<String, Object>());
+    private static final Map<String, Object> EMPTY_MAP = Collections.unmodifiableMap(new HashMap<>());
```

Comment:
minor nit, we could probably just use `Collections.emptyMap()` or `ImmutableMap.of()` here.